gnomad_qc/v3/resources/annotations.py

```diff
@@ -1,5 +1,9 @@
-from gnomad.resources.resource_utils import (TableResource,
-                                             VersionedTableResource)
+from typing import Optional
+
+from gnomad.resources.resource_utils import (DataException,
```

Comment:
We don't currently have Black formatting enforced in PR checks for this repository. It looks like we never reformatted this code like gnomad_methods, since `black --check` says 47 of 56 files would be reformatted.

Reply:
Looks like it hasn't been run, but I'll run it for this commit